inc/classes/class_db_mysql.php

```diff
@@ -2,48 +2,30 @@
 
 class db
 {
-    public $mysqli = 0;
     public $link_id = 0;
     public $query_id = 0;
-    public $record   = array();
-    public $print_sql_error;
+    public $record = array();
     public $success = false;
     public $count_query = 0;
     public $errors = '';
     public $errorsFound = 0;
     public $connectfailure = 0;  //0= no error, 1=connection error, 2=database error
     public $QueryArgs = array();
 
-  // Construktor
-    public function __construct()
```

Comment:
Gave it a try, no credentials are leaked if the connection attempt fails.
Error message thrown is:

`Das MySQL-PHP Modul ist nicht geladen. Bitte füge die mysql.so Erweiterung zur php.ini hinzu und restarte den Webserver neu. Lansuite wird abgebrochen
Fatal error: Uncaught Error: Call to undefined function mysql_connect() in C:\xampp\htdocs\lansuite\inc\classes\class_db_mysql.php:65 Stack trace: #0 C:\xampp\htdocs\lansuite\index.php(210): db->connect(1) #1 {main} thrown in C:\xampp\htdocs\lansuite\inc\classes\class_db_mysql.php on line 65`